packages/editor/src/components/collab-sidebar/add-comment.js

```diff
@@ -47,7 +47,8 @@ export function AddComment( {
 			className="editor-collab-sidebar-panel__thread is-selected"
 			spacing="3"
 			tabIndex={ 0 }
-			role="listitem"
+			role="treeitem"
+			aria-label={ __( 'New Note' ) }
```

Comment:
This is a small bug fix; When a first note is being added to the block, it lacks a label, even though the thread is focusable.